packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/OrderCheckout.js

```diff
@@ -26,11 +26,7 @@ const OrderCheckout = ({ quoteR, rateQuoteR, account, onFetchQuote, reason, limi
   }
 
   const rateHelper = () => {
-    let quote
-    if (Remote.NotAsked.is(quoteR)) quote = rateQuoteR
-    else quote = quoteR
-
-    return quote.map(q => {
+    return rateQuoteR.map(q => {
```

Comment:
I don't get this change. We don't want to get the rate from the `rateQuote` if a user fetched `quote` is available. That's what led to the difference in rate and quoted buy/sell amount. Am I missing something?

Reply:
The problem I'm solving here is that if you go to the order review page and then click on buy or sell to return the checkout screen you will see the same rate in buy and sell which is the rate of the quote you obtained whereas the two should be different